src/Admin/BaseMediaAdmin.php

```diff
@@ -115,6 +115,8 @@ protected function configurePersistentParameters(): array
             $context = $request->query->get('context', $this->pool->getDefaultContext());
         }
 
+        \assert(\is_string($context));
```

Comment:
Not sure if writing `$request->query->getString` solve the issue

Reply:
I think it would resolve the issue, but the method `getString()` is not available before "symfony/http-foundation" 6.3.
See https://github.com/symfony/http-foundation/blob/7.0/CHANGELOG.md#63.

Reply:
Oh... :(